Src/library/cURL/curl_opt_constants.e

```diff
@@ -801,6 +801,16 @@ feature -- Protocol
 			is_class: class
 		end
 
+	curlopt_mimepost: INTEGER
+			-- Declared as CURLOPT_TIMEVALUE
```

Comment:
the comment mentions CURLOPT_TIMEVALUE ... but I guess this is a copy/paste error as it should mention CURLOPT_MIMEPOST

Reply:
yes